less/plugins/adapt-contrib-media/mep.less

```diff
@@ -0,0 +1,237 @@
+    background: none;
+    background-color: @mejs-time-total;
+  }
+
+  .mejs-controls .mejs-time-rail .mejs-time-loaded {
+    background: none;
+    background-color: @mejs-time-loaded;
+  }
+
+  .mejs-controls .mejs-time-rail .mejs-time-current,
+  .mejs-controls .mejs-volume-button .mejs-volume-slider .mejs-volume-current {
+    background: none;
+    background-color: @mejs-time-current;
+  }
+
+  .mejs-controls .mejs-captions-button .mejs-captions-selector,
+  .mejs-controls .mejs-volume-button .mejs-volume-slider {
+    background: none;
+    background-color: @mejs-color;
+  }
+
+  .mejs-controls .mejs-volume-button .mejs-volume-slider .mejs-volume-handle {
+    background: none;
+    background-color: @mejs-time-current;
+  }
+
+  // Icon overrides
+  // icon classes must be defined in core/fonts/vanilla or
+  // a custom font must be set up in the theme
+  // --------------------------------------------------
+  .mejs-controls .mejs-play button {
+    .icon-video-play;
+  }
+
+  .mejs-controls .mejs-pause button {
+    .icon-video-pause;
+  }
+
+  .mejs-controls .mejs-fullscreen-button button {
+    .icon-video-fullscreen;
+  }
+
+  .mejs-controls .mejs-unfullscreen button {
+    .icon-video-exit-fullscreen;
+  }
+
+  .mejs-controls .mejs-mute button {
+    .icon-sound;
+  }
+
+  .mejs-controls .mejs-unmute button {
+    .icon-sound-mute;
+  }
+
+  .mejs-controls .mejs-captions-button button {
+    .icon-video-captions-off-2;
+  }
+
+  .mejs-controls .mejs-captions-enabled button {
+    .icon-video-captions;
+  }
+
+  // Player controls
+  // --------------------------------------------------
+  .mejs-controls {
+    height: @mejs-controls-height;
+  }
+
+  .mejs-controls .mejs-time-rail {
+    height: @mejs-time-rail-height;
+    margin: ((@mejs-controls-height - @mejs-time-rail-height) / 2) @mejs-margin;
+    padding: 0;
+  }
+
+  .mejs-controls .mejs-time-rail span {
+    height: @mejs-time-rail-height;
+  }
+
+  .mejs-controls .mejs-time-rail .mejs-time-total {
+    height: @mejs-time-rail-height;
+    margin: 0;
+  }
+
+  .mejs-controls .mejs-time {
+    height: @mejs-time-size;
+    margin: ((@mejs-time-size / 2) + (@mejs-margin / 2)) @mejs-margin;
+    padding: 0;
+    font-size: @mejs-time-size;
+    line-height: 1;
+  }
+
+  // Button icon style overrides
+  // --------------------------------------------------
+  .mejs-controls .mejs-button {
+    height: @icon-size;
+    width: @icon-size;
+    margin: @mejs-margin / 2;
+  }
+
+  .mejs-controls .mejs-button button {
+    .icon;
+    height: @icon-size;
+    width: @icon-size;
+    margin: 0;
+    background: none;
+  }
+
+  // Current time pop float on time rail hover
+  // --------------------------------------------------
+  .mejs-controls .mejs-time-rail .mejs-time-float {
+    top: -1.5rem;
+    height: calc(~'@{mejs-time-float-size} + 2px');
+    width: calc(~'@{mejs-time-float-width} + 2px');
```

Comment:
Not in this instance as it is adding the border width (2 because it's adding the top and bottom values / left and right values) defined in the mep.less that comes with the media component: https://github.com/adaptlearning/adapt-contrib-media/blob/master/less/mep.less#L348